### PR TITLE
Lib folder

### LINKED ISSUE #21: Repository folder structure
There are a lot of files on the top level of this repo. It gets confusing pretty quick. I made a table on the readme that clarifies what is what but on a quick glance it should be easier to understand the structure and what to open. Namely I first propose to move every max abstraction into a subfolder. The easier way to do that and have max still be aware of it, is to make use of max projects. This in turn will be helpful also for the upcoming example patches being develop on the [CreativeExamples](https://github.com/omarcostahamido/PVM/tree/CreativeExamples) branch.

### PR DESCRIPTION
**Pull request by omarcostahamido**
- this fixes #21 
- users will only have to open `pvm.maxproj` now

### CODE REVIEW

File: lib/pvm.maxpat [hunk 1]
```diff
@@ -566,36 +566,36 @@
 		}
 ,
 		"dependency_cache" : [ 			{
-				"name" : "pvm_send.maxpat",
-				"bootpath" : "~/Downloads/PVM",
-				"patcherrelativepath" : ".",
-				"type" : "JSON",
+				"name" : "max-init.txt",
+				"bootpath" : "~/Documents/GitHub/PVM",
```
Comment (KaneBetter) [inline lib/pvm.maxpat:570]: What does this path mean, other than LGTM

Comment (omarcostahamido) [reply lib/pvm.maxpat:570]: max projects creates those paths for keeping track of the related files. I think these are not set in stone, and that max is clever enough to realize when it is on a different machine. Still, pretty weird that it is not a relative path. I can try to open this branch on another machine to double check at some point. Or maybe you could try it already on the studio iMac? thanks

Comment (KaneBetter) [reply lib/pvm.maxpat:570]: OK, I see, then everything looks good.

**Review by KaneBetter — APPROVED** [review]
LGTM

File: lib/pvm.maxpat (hunk 1 above)
Comment (omarcostahamido) [reply lib/pvm.maxpat:570]: yes, just confirmed this. if you look at the line right after that it has the relative path. So, we're good!

**Pull request merged**